modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/DateTimeObjectFieldBusinessType.java

```diff
@@ -94,6 +95,35 @@ public String getLabel(Locale locale) {
 		return _language.get(locale, "date-and-time");
 	}
 
+	@Override
+	public Map<Locale, Object> getLocalizedValue(
+			ObjectField objectField, Long userId, Map<String, Object> values)
+		throws PortalException {
+
+		Map<Locale, Object> localizedValues =
+			ObjectFieldBusinessType.super.getLocalizedValue(
+				objectField, userId, values);
+
+		if (localizedValues == null) {
+			return null;
+		}
+
+		String sourceTimeZoneId = ObjectFieldSettingUtil.getTimeZoneId(
+			objectField.getObjectFieldSettings(),
+			_userLocalService.getUser(userId));
```

Comment:
Can you inline this variable, please?

Reply:
Done! I made a small change in this file can you review again please ?